Intermediate Route Fixes (small edit to advanced route)

SheepofRootSquare opened the pull request:
Intermediate route reformatted a bit. Less Redundancy more clarity. 
Advanced Route Fly Menu fixed. Advanced Route Lorelei Updated to perform swap before fight (assuming no yolo).

SheepofRootSquare committed:
Intermediate Route Fixes

Fixed Links. Clarification here and there. Removed some redundancy.

SheepofRootSquare committed:
Fly Menu & Lorelei

Fixed wording (super repel, not repel) and updated non-yolo Lorelei to perform a swap before entering the fight.

SheepofRootSquare committed:
Merge pull request #1 from SheepofRootSquare/SheepofRootSquare-patch-1

Intermediate Route Fixes

TiKevin83 commented on docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md:
```diff
@@ -1,10 +1,9 @@
-> Note: The above document is useful, but contains some higher risks than this route suggests.
-- [Defensive Damage Ranges](../../../resources/defensive-ranges.md)
-- [Offensive Damage Ranges](../../../resources/offensive-ranges.md)
+- [Defensive Damage Ranges](https://pokemon-speedrunning.github.io/speedrun-routes/#/gen-1/red-blue/main-glitchless/resources/defensive-ranges)
```
these should still be relative URLs but I think you're correct to drop .md

SheepofRootSquare committed:
Fixed links w/in github

SheepofRootSquare committed:
2 dots for resource links

SheepofRootSquare committed:
extra parentheses removed

SheepofRootSquare committed:
Clarifying that old Lorelei Strat is still valid

SheepofRootSquare committed:
Clarifying that old Lorelei strat is still fine

SheepofRootSquare committed:
Revert Intermediate Back to Old Lorelei

SheepofRootSquare committed:
Last update - Lorelei

SheepofRootSquare committed:
Last Lorelei update - Intermediate Lorelei

SheepofRootSquare committed:
Update no lass

Swap to bird if 2 sand on Bridge Rival, X-Special stall Weezing; updated Backup Strats to be paste with explanations of what to do provided; Update Viridian Rival to only use on X-Special instead of two.

SheepofRootSquare committed:
update Lass

Same as no lass changes

SheepofRootSquare committed:
lass

intermediate backup pastebin added after koga

SheepofRootSquare committed:
Shorts guy accidentally dropped lead HA strat

SheepofRootSquare committed:
surf menu slot 3 not 2 for srepel
